docs/pages/access-controls/device-trust/enforcing-device-trust.mdx

```diff
@@ -155,13 +150,19 @@ leaf clusters.
 ## App Access support
 
 Apps may enforce device trust via [role-based enforcement](
-#role-based-trusted-device-enforcement). Any apps that enforce device trust must
-be accessed using tsh commands such as [tsh proxy app](
-../../reference/cli/tsh.mdx#tsh-proxy-app) or using the certificates issued by
-`tsh app login`. Direct access to those apps via the Web UI is not yet
-available.
+#role-based-trusted-device-enforcement).
+
+To access apps protected by device trust using the Web UI, make sure your device
+is [registered and enrolled](
+./device-management.mdx#register-a-trusted-device), install [Teleport Connect](
```

Comment:
Should we say v16 or later here?

Reply:
4cb30e1049

Mentioned here and in App Access troubleshooting, but kept other sections as-is.